packages/icons/src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import { Icon } from '../';
+
+function App() {
+  return (
+    <>
+      <Icon name="WoocommerceColored" size="xlarge" customColor="#0000ff" />
+      <div>Hello icon components library!</div>
```

Comment:
I would improve this example if possible. It should help testing any changes to this library.